contrib/externs/angular-1.5.js

```diff
@@ -717,6 +735,15 @@ angular.Module = function() {};
 angular.Module.prototype.animation = function(name, animationFactory) {};
 
 /**
+ * Intended to be called with a name string and a component definition.
+ *
+ * @param {string} name
+ * @param {Object<string, *>} componentDefinition
```

Comment:
Unfortunately, using `!angular.Component` would require a cast in user code to handle [optional properties](https://groups.google.com/forum/#!topic/closure-compiler-discuss/3qhcDfa9uX4) or to handle `"custom properties"` (`$... – {function()=} – additional annotations to provide to the directive factory function.`).

Directives are handled with a `angular.Injectable=` param which is also a loose type.

I added the missing `!`; if you still prefer `!angular.Component` I can update the PR.